rstar/Cargo.toml

```diff
@@ -8,14 +8,15 @@ repository = "https://github.com/georust/rstar"
 license = "MIT/Apache-2.0"
 readme = "README.md"
 edition = "2018"
+rust-version = "1.51"
 keywords = ["rtree", "r-tree", "rstar", "spatial", "nearest-neighbor"]
 categories = ["data-structures", "algorithms"]
 
 [badges]
 maintenance = { status = "actively-developed" }
 
 [dependencies]
-heapless = "0.6"
+heapless = "0.7"
```

Comment:
Could you specify the full version you've tested against here?

see https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277 for rationale.

It's something I've only recently realized that we've gotten wrong historically, but I'd like to get on track with it.

Reply:
Sure! I was initially thinking of including the full version but when I saw it was just specified to two components I leaned towards keeping it consistent.

Since I tested against v0.7.10, I put that here.